2013/data_file_specification.json

```diff
@@ -260,6 +390,9 @@
                "end": "54",
                "length": "5",
                "dataType": "N",
+               "format": "currency,thousands",
```

Comment:
Consistency of the value. By mixing in an array, you would have to do type checking on `format` during implementation. Is it a string? Is it an array? Either that, or you would have to make them all array, again to maintain the consistency.